docs/en/upgrade.md

```diff
@@ -2,8 +2,12 @@
 
 !!! warning "Important"
 
-    1. Please be advised that upgrading to this version directly from version 1.0.x is not supported. It is necessary to upgrade to [version 1.1.5][v115] first. 
-    2. Upgrading from version 1.1.5 or any earlier 1.1 version will rebuild the default analysis dashboard and you will not be able to analyze previous data. If you need to migrate the existing data, it is imperative that you contact the AWS Clickstream team for support through your sales representative.
+    1. Please be advised that upgrading directly from version 1.0.x to this version is not supported. It is necessary to upgrade to [version 1.1.5][v115] first.
+    2. By upgrading the web console from earlier 1.1 versions before 1.1.6, you could continue to view the dashboards of the project. However, you could not [explore the existing Clickstream data][exploration] due to the feature relying on the updated data schema.
```

Comment:
However, you could not [explore the existing Clickstream data][exploration] due the changes of data schemas. If you wish to continue to use the Explorations, you will need to upgrade the data pipeline as well and migrate the existing data to new data schemas (if you want to explore historical data).